uyuni/common-libs/common/rhn_rpm.py

```diff
@@ -47,6 +47,7 @@
 
 # need this for rpm-pyhon < 4.6 (e.g. on RHEL5)
 rpm.RPMTAG_FILEDIGESTALGO = 5011
+rpm.RPMTAG_MODULARITYLABEL = 5096
```

Comment:
You are missing an entry at `uyuni-common-libs.changes`

Reply:
ah, different packages, right.

Reply:
done